packages/yew-macro/tests/function_component_attr/with-defaulted-type-param-pass.rs

```diff
@@ -1,4 +1,3 @@
-#![no_implicit_prelude]
```

Comment:
I removed these from the tests that has `#[derive(Properties)]` because it prevents `#[diagnostics]` from being used. I have no clue how to qualify diagnostics with `no_implicit_prelude`. If anybody can help it's very appreciated.

Reply:
Hmm.. I think  it's impossible to use `no_implicit_prelude` together with `diagnostic`

https://doc.rust-lang.org/reference/attributes.html#r-attributes.tool.prelude

> Tool attributes are not available if the [no_implicit_prelude](https://doc.rust-lang.org/reference/names/preludes.html#the-no_implicit_prelude-attribute) attribute is used.

Reply:
I think no_implicit_prelude was introduced to improve hygiene, it ensures our macros don't accidentally collide with rust's built-in ones, which could cause false passes in macro tests. 

Sadly there seems to be no way around it for now and diagnostic improvements seem worth it.